common/presenters/move-to-meta-list-component.js

```diff
@@ -79,6 +81,9 @@ function moveToMetaListComponent(move, journeys, { updateUrls = {} } = {}) {
       })}`
     : journeysSummary.map(({ toLocation }) => toLocation).join(' and ')
 
+  const journeyDates = journeysSummary.map(({ date }) => date).join(' to ')
```

Comment:
As discussed, we should probably just do "{first journey date} to {last journey date}", but I'm going to address that under MAP-510 anyway

Reply:
👍